libs/community/langchain_community/vectorstores/lancedb.py

```diff
@@ -55,17 +66,34 @@ def __init__(
         api_key: Optional[str] = None,
         region: Optional[str] = None,
         mode: Optional[str] = "overwrite",
+        table: Optional[Any] = None,
```

Comment:
```suggestion
        *,
        table: Optional[Any] = None,
```
make them kwarg-only

Reply:
do you mean to make the cloud connection components kwarg only or all arguments as kwargs ? any specific reason why ?
thanks